README.md

```diff
@@ -29,17 +29,15 @@ will need to support both published releases and [pre-release versions
 (#12)](https://github.com/openshift-kni/install-scripts/issues/12) of
 each of these.
 
-The scripts will:
+To ease installtion, a [prepared ISO](https://github.com/openshift-kni/install-scripts/issues/20)
+can be used to install the "provioning host".  Using the ISO will address
```

Comment:
"Using the ISO will address " - change to: "Using the prepared ISO addresses the following:" , then list out everything. At the bottom right a note. "NOTE: Optional scripts may be executed if not using the prepared ISO that handle the above prerequisites."